test/unit/proxy_test.rb

```diff
@@ -478,17 +478,6 @@ def test_apicast_configuration_driven
     end
   end
 
-  test 'save_and_deploy' do
```

Comment:
Do we have another test for apicast v2 or service mesh?

Reply:
no there is not.